Gemfile

```diff
@@ -130,4 +130,8 @@ group :development do
   gem 'guard-rspec'
   gem 'parallel_tests'
   gem 'rubocop', '>= 0.49.1'
+
+  # We don't actually require this gem directly, but 1.0.9 fixed openssl issues
+  # on macOS: https://github.com/eventmachine/eventmachine/issues/602
+  gem 'eventmachine', '>= 1.0.9'
```

Comment:
Sorry to be a PITA, but I'm just curious. Why we can't use `gem 'eventmachine', '>= 1.2.3'`. Is there some dependency deep in the tree that is blocking us?